images/cluster-capacity/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#
+# This is the cluster capacity tool.
+#
+# The standard name for this image is openshift/origin-cluster-capacity
+#
+FROM centos:centos7
+
+COPY bin/hypercc /bin/hypercc
```

Comment:
how would that work who wanted to build images from local sources like OS_RELEASE=n ./hack/build-images.sh?